### PR TITLE
PMM-9669 IA Template Copy and New Secondary Labels QAA Testcase

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
added test for case T1137 Verify that IA alerts are showing important labels first 
and test for case T1116 Verify user is able to copy alert rule, source template of which was deleted

https://pmm.cd.percona.com/view/all/job/pmm2-ui-tests/13037/
image

### CODE REVIEW

File: tests/ia/alerts_test.js [hunk 1]
```diff
@@ -152,6 +152,24 @@ Scenario(
...
 
+Scenario(
+  'PMM-T1137 Verify that IA alerts are showing important labels first @ia @alyona-p',
+  async ({ I, alertsPage }) => {
+    I.amOnPage(alertsPage.url);
+    I.waitForElement(alertsPage.elements.alertRow(alertName), 30);
+    I.click(alertsPage.buttons.arrowIcon(alertName));
+    I.waitForVisible(alertsPage.elements.details, 30);
+    I.seeElement(alertsPage.elements.detailsRuleExpression, 30);
+    I.seeElement(alertsPage.elements.detailsSecondaryLabels, 30);
```
Comment (puneet0191) [inline tests/ia/alerts_test.js:163]: where do you verify important labels exist?

**Review by puneet0191 — DISMISSED** [review]

**Review by puneet0191 — DISMISSED** [review]

File: tests/ia/pages/alertsPage.js [hunk 2]
```diff
@@ -108,4 +114,11 @@ module.exports = {
       assert.equal(warningColor, this.colors.warning, 'Warning alert is silence');
     }
   },
+
+  checkContainingLabels(expectedLabels, alertName = null) {
```
Comment (yurkovychv) [inline tests/ia/pages/alertsPage.js:118]: please change this method to accept object with keys: `primaryLabels`, `secondaryLabels` and `alertName`

File: tests/ia/pages/alertsPage.js [hunk 3]
```diff
@@ -2,27 +2,33 @@ const { I } = inject();
 const assert = require('assert');
 
 const alertRow = (alertName) => `//tr[td[contains(., "${alertName}")]]`;
+const details = locate('$alert-details-wrapper');
```
Comment (yurkovychv) [inline tests/ia/pages/alertsPage.js:5]:
```suggestion
const details = '$alert-details-wrapper';
```

File: tests/ia/pages/alertsPage.js [hunk 4]
```diff
@@ -2,27 +2,33 @@ const { I } = inject();
...
+    labelsCell: (alertName, text) => locate(`${alertRow(alertName)}/td[4]`).find('$chip').withText(text),
     stateCell: (alertName) => `${alertRow(alertName)}/td[3]`,
     severityCell: (alertName) => `${alertRow(alertName)}/td[2]`,
     criticalSeverity: '//td[2]/span[text()="Critical"]',
     highSeverity: '//td[2]/span[text()="High"]',
     noticeSeverity: '//td[2]/span[text()="Notice"]',
     warningSeverity: '//td[2]/span[text()="Warning"]',
     columnHeaderLocator: (columnHeaderText) => `//th[text()="${columnHeaderText}"]`,
+    details,
+    detailsRuleExpression: locate(`${details}`).find('div').withText('Rule Expression'),
```
Comment (yurkovychv) [inline tests/ia/pages/alertsPage.js:22]:
```suggestion
    detailsRuleExpression: locate(details).find('div').withText('Rule Expression'),
```

Comment (yurkovychv) [reply tests/ia/pages/alertsPage.js:22]: same in other places

File: tests/ia/pages/alertsPage.js [hunk 5]
```diff
@@ -2,27 +2,33 @@ const { I } = inject();
...
+const details = locate('$alert-details-wrapper');
 
 module.exports = {
   url: 'graph/integrated-alerting/alerts',
   columnHeaders: ['Name', 'Severity', 'State', 'Labels', 'Active Since', 'Last Notified', 'Actions'],
   elements: {
     noData: '$table-no-data',
     alertRow: (alertName) => alertRow(alertName),
-    labelsCell: (alertName) => `${alertRow(alertName)}/td[4]//span`,
+    labelsCell: (alertName, text) => locate(`${alertRow(alertName)}/td[4]`).find('$chip').withText(text),
```
Comment (yurkovychv) [inline tests/ia/pages/alertsPage.js:13]: why did you change `labelsCell` locator? it is not `labelsCell` anymore in your locator. It is a `primaryLabel` locator

**Review by yurkovychv — CHANGES_REQUESTED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**